src/app/components/Validator/components/ValidatorOperators.tsx

```diff
@@ -143,9 +148,11 @@ const ValidatorOperators = (props: ValidatorOperatorProps) => {
                   <StyledCell key="operator-performance">
                     <Skeleton />
                   </StyledCell>
+                  {!isMobileDevice && (
                   <StyledCell key="operator-status">
                     <Skeleton />
                   </StyledCell>
+)}
```

Comment:
Reformat the file to have proper indentation
Check in your IDE settings to run `eslint --fix` when you saving files, or run it manually on all the project every time before commit